app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -46,7 +43,7 @@
     {{/ui-dropdown}}
   </div>
   <div class="ui buttons">
-    <button class="ui teal button" type="submit">{{t 'Update Page'}}</button>
-    <button class="ui red button">{{t 'Delete Page'}}</button>
+    <button class="ui teal button" type="submit">{{if isCreate 'Save' 'Update'}}</button>
+    <button class="ui red button" {{action 'deletePage' data}}>{{if isCreate 'Cancel' 'Delete'}}</button>
```

Comment:
Always ask a confirmation for all delete actions. Use the `confirm` helper for it.